Лабораторная работа 2/task_2.py

```diff
@@ -0,0 +1,17 @@
+def find_common_participants(group1, group2, limiter=','):
+    group1 = group1.split(limiter)
+    group2 = group2.split(limiter)
+    common_participants =\
+        list(set(group1).intersection(group2))
+    common_participants.sort()
+# TODO Напишите функцию find_common_participants
+    return common_participants
+
+
+participants_first_group = "Иванов|Петров|Сидоров"
+participants_second_group = "Петров|Сидоров|Смирнов"
+
+participants_intersection =\
+    find_common_participants(participants_first_group, participants_second_group, delimiter='|')
```

Comment:
Отличное решение!
Но вы допустили небольшую ошибку.
```suggestion
    find_common_participants(participants_first_group, participants_second_group, limiter='|')
```